crates/oxc_minifier/src/peephole/replace_known_methods.rs

```diff
@@ -525,6 +527,18 @@ impl<'a> PeepholeOptimizations {
             "sign" if arg_val.to_bits() == 0f64.to_bits() => 0f64,
             "sign" if arg_val.to_bits() == (-0f64).to_bits() => -0f64,
             "sign" => arg_val.signum(),
+            #[expect(clippy::cast_sign_loss)]
+            #[expect(clippy::cast_possible_truncation)]
+            "clz32" => {
+                let modulus = 2f64.powi(32);
+                let base = if arg_val < 0f64 {
+                    let intermediate = arg_val % modulus;
+                    if intermediate < 0f64 { intermediate + modulus } else { intermediate }
+                } else {
+                    arg_val % modulus
+                } as u32;
```

Comment:
I guess this is equivalent to `(arg_val.to_int_32() as u32) & 31`.
https://github.com/oxc-project/oxc/blob/df68910f8be455167ce04b30207a0b5e9c6133dc/crates/oxc_ecmascript/src/constant_evaluation/mod.rs#L272
I think we should add `.to_uint_32`.